Dockerfile

```diff
@@ -69,26 +87,43 @@ RUN ${CONDA_EXECUTABLE} env create -f ./danesfield/deployment/conda/conda_env.ym
 
 # Install core3d-tf_ops package from kitware-danesfield / defaults
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
-      conda clean -tipsy"]
+  conda activate core3d && \
+  pip install pip==20.0.1 && \
+  conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
+  conda clean -tipsy"]
 
 # Install opencv package from conda-forge
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c conda-forge -y opencv && \
-      conda clean -tipsy"]
+  conda activate core3d && \
+  conda install -c conda-forge -y opencv && \
+  conda clean -tipsy"]
+
+RUN chmod +x pipeline.sh && \
```

Comment:
To be consistent, add a comment here like
```
# Install ColmapForVisSat package from Github
```
Then move the `git clone` call after that comment and before this install statement.

Reply:
You may actually want to combine the `git clone` and install into one `RUN` command.